cetmix_tower_server/models/cx_tower_jet_template_install.py

```
@@ -0,0 +1,248 @@
+            # Run the corresponding flight plan
+            if flight_plan:
+                # Update the current template install line
+                self.write(
+                    {
+                        "current_line_id": installation_task.id,
+                    }
+                )
+                # WARNING: Explicit commit to ensure visibility across transactions
+                # This prevents race conditions in async flight plan callbacks
+                if not self.env.context.get("no_transaction_commit"):
+                    self.env.cr.commit()  # pylint: disable=invalid-commit
```

Pylint (3.3.9)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'invalid-commit'

(W0012)